molecule/default/molecule.yml

```diff
@@ -2,7 +2,8 @@
 dependency:
   name: galaxy
   options:
-    role-file: requirements.yml
+    role-file: molecule_requirements.yml
+    requirements-file: molecule_requirements.yml
```

Comment:
Why both role-file and requirements-file?

Reply:
Ive removed those both additions and reverse engineered molecule so i know what requirement files it actually needs.

role requirements -> `molecule/default/requirements.yml`
collection requirements -> `molecule/default/collections.yml`